src/composables/useKeyboardShortcuts.ts

```diff
@@ -1,34 +1,25 @@
-import { onKeyDown } from '@vueuse/core';
+import { ref, watch } from 'vue';
+import { useMagicKeys, whenever } from '@vueuse/core';
 
-import { DECREMENT_LABEL_KEY, INCREMENT_LABEL_KEY } from '../config';
-import { useToolStore } from '../store/tools';
-import { Tools } from '../store/tools/types';
-import { useRectangleStore } from '../store/tools/rectangles';
-import { useRulerStore } from '../store/tools/rulers';
-import { usePolygonStore } from '../store/tools/polygons';
+import { getEntries } from '../utils';
+import { ACTION_TO_KEY } from '../config';
+import { ACTION_TO_FUNC } from './actions';
 
-const applyLabelOffset = (offset: number) => {
-  const toolToStore = {
-    [Tools.Rectangle]: useRectangleStore(),
-    [Tools.Ruler]: useRulerStore(),
-    [Tools.Polygon]: usePolygonStore(),
-  };
-  const toolStore = useToolStore();
+export const actionToKey = ref(ACTION_TO_KEY);
 
-  // @ts-ignore - map may not have all keys of tools
-  const activeToolStore = toolToStore[toolStore.currentTool];
-  if (!activeToolStore) return;
-
-  const labels = Object.entries(activeToolStore.labels);
-  const activeLabelIndex = labels.findIndex(
-    ([name]) => name === activeToolStore.activeLabel
-  );
+export function useKeyboardShortcuts() {
+  const keys = useMagicKeys();
+  const unwatchFuncs = ref([] as Array<ReturnType<typeof whenever>>);
```

Comment:
`unwatchFuncs` has no reason to be a ref.

Reply:
Yes!  Fixed.